1s
Current runner version: '2.321.0'
Operating System
Runner Image
Runner Image Provisioner
GITHUB_TOKEN Permissions
Secret source: Actions
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v3' (SHA:f43a0e5ff2bd294095638e18286ca9a3d1956744)
Download action repository 'actions/setup-python@v4' (SHA:65d7f2d534ac1bc67fcd62888c5f4f3d2cb2b236)
Download action repository 'codecov/codecov-action@v3' (SHA:ab904c41d6ece82784817410c45d8b8c02684457)
Complete job name: test
20s
Checking docker version
Clean up resources from previous jobs
Create local container network
Starting postgres service container
Waiting for all services to be ready
1s
Run actions/checkout@v3
Syncing repository: Kaiser9005/ERP
Getting Git version info
Temporarily overriding HOME='/home/runner/work/_temp/4775e81a-728f-455f-b2be-d4b35e91591a' before making global git config changes
Adding repository directory to the temporary git global config as a safe directory
/usr/bin/git config --global --add safe.directory /home/runner/work/ERP/ERP
Deleting the contents of '/home/runner/work/ERP/ERP'
Initializing the repository
Disabling automatic garbage collection
Setting up auth
Fetching the repository
Determining the checkout info
Checking out the ref
/usr/bin/git log -1 --format='%H'
'6bdabaf841ba3ced2f052c0763c364d247d0ca13'
0s
Run actions/setup-python@v4
Installed versions
22s
1s
Run pytest --cov=. --cov-report=xml
ImportError while loading conftest '/home/runner/work/ERP/ERP/tests/conftest.py'.
tests/conftest.py:16: in <module>
    from main import app
main.py:5: in <module>
    from api.v1 import api_router
api/v1/__init__.py:2: in <module>
    from .endpoints import (
api/v1/endpoints/production.py:4: in <module>
    from ....db.database import get_db
E   ImportError: attempted relative import beyond top-level package
Error: Process completed with exit code 4.